algorithmic_efficiency/workloads/librispeech_conformer/librispeech_pytorch/workload.py

```diff
@@ -253,7 +253,7 @@ def _eval_model_on_split(self,
     data_rng, model_rng = prng.split(rng, 2)
     if split not in self._eval_iters:
       # These iterators repeat indefinitely.
-      self._eval_iters[split] = itertools.cycle(
+      self._eval_iters[split] = (
           self._build_input_queue(
```

Comment:
Nit: shift this to the line above.

Reply:
Because we have our own BatchNorm implementation that takes padding as input and tracks stats. 
Yes, it does nothing for the other parameters.